deepinfra/utils/form_data.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from requests_toolbelt import MultipartEncoder
+
+from deepinfra.utils.read_stream import ReadStreamUtils
+
+
+class FormDataUtils:
+	"""
+	Utilities for creating form data.
+
+	"""
+	@staticmethod
+	def get_form_data(data, blob_keys=()):
+		"""
+		Creates a MultipartEncoder object from the data.
+		:param data:
+		:param blob_keys:
+		:return:
+		"""
+		body = {}
+
+		for key, value in data.items():
+			if key in blob_keys:
+				body[key] = (key,ReadStreamUtils.get_read_stream(value))
```

Comment:
space after `,`. You can setup some auto-formatter afterwords (black I think is used for py)

Reply:
Opened this as an issue: https://github.com/ovuruska/deepinfra-python/issues/7